pkgs/development/python-modules/pyemvue/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+# build-system
+, hatchling
+, build
+
+# propagated modules
+, requests
+, dateutil
+, python-jose
+, pycognito
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "pyemvue";
+  version = "0.18.0";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```

we changed that slightly recently but behind the backs it functions the same